Rails 5

peggles2 opened the pull request:
Upgrade ruby on rails to version 5.0

MothOnMars commented:
@peggles2 ,  I noticed that https://github.com/GSA/search-gov/pull/326 was merged without any of the requested changes. Can you open a new PR to fix those issues in the release branch?

peggles2 commented:
@MothOnMars looks like https://github.com/GSA/search-gov/pull/327/files addressed all the comments in 327 and was approved. I will make note in that pull request as well. Thanks!
